<?xml version="1.0"?>

<launch>
    <!-- <arg name="param_file_name" default="$(find pointcloud_processing)/config/param.yaml"/> -->
    <arg name="darknet_topic" default="darknet_ros/boundingbox"/>
    <arg name="pcl_topic" default="camera/depth/color/points"/>
    <arg name="target_frame" default="map"/>
    <arg name="visual" default="true"/>
    <arg name="pcl_visual" default="true"/>

    <node pkg="pointcloud_processing" type="pcl_processing" name="localnavi_server" output="screen">
        <!-- <rosparam command="load" file="$(arg param_file_name)"/> -->
        <param name="DARKNET_TOPIC" value="$(arg darknet_topic)"/>
        <param name="PCL_TOPIC" value="$(arg pcl_topic)"/>
        <param name="TARGET_FRAME" value="$(arg target_frame)"/>
        <param name="VISUAL" value="$(arg visual)"/>
        <param name="PCL_VISUAL" value="$(arg pcl_visual)"/>
    </node>
</launch>
